conf/application.conf

```diff
@@ -95,7 +95,7 @@ metrics {
 }
 
 auditing {
-  enabled = false
```

Comment:
Do we want this enable on local?
Probably we could disable here and the `base.conf` will enable it for all the other environments

Reply:
reverted back to false